app/helpers/application_helper.rb

```diff
@@ -131,12 +133,12 @@ def follow_button(followable, style = "full", classes = "", aria_label: "")
       data: {
         info: {
           id: followable.id,
-          className: followable.class.name,
+          className: followable_type,
+          name: followable_name,
           style: style
         }
       },
       class: "crayons-btn follow-action-button whitespace-nowrap #{classes} #{user_follow}",
-      aria: { label: aria_label },
```

Comment:
I think it would make sense to continue setting the `aria-label` here too, e.g.

```
aria: { label: "Follow #{followable_type}: #{followable_name}" },
```

Then the aria-label is available on initial page load, even before any of the JS is run.